core/store/migrate/migrations/0141_allow_broadcast_at_on_fatally_errored_tx.sql

```diff
@@ -0,0 +1,32 @@
+-- +goose Up
+ALTER TABLE eth_txes DROP CONSTRAINT chk_eth_txes_fsm;
+ALTER TABLE eth_txes ADD CONSTRAINT chk_eth_txes_fsm CHECK (
+    state = 'unstarted'::eth_txes_state AND nonce IS NULL AND error IS NULL AND broadcast_at IS NULL AND initial_broadcast_at IS NULL
+    OR
+    state = 'in_progress'::eth_txes_state AND nonce IS NOT NULL AND error IS NULL AND broadcast_at IS NULL AND initial_broadcast_at IS NULL
+    OR
+    state = 'fatal_error'::eth_txes_state AND nonce IS NULL AND error IS NOT NULL
```

Comment:
This is the only one that changed - relaxed requirement to have broadcast_at and initial_broadcast_at be nil